app/components/post_textbox/post_textbox.js

```diff
@@ -14,6 +14,7 @@ import PaperPlane from 'app/components/paper_plane';
 import {changeOpacity, makeStyleSheetFromTheme} from 'app/utils/theme';
 
 import Typing from './components/typing';
+import {removeAscii} from "../../utils/url";
 
```

Comment:
Well it is specifically a util method for encoding a url. I am not sure why else it would be used and I assumed we don't like throwing everything in general just because it could be general; that would make everything general?
But I am 0/5 on where it goes so I will move it there.